cypress/integration/frontsite/literatureDetails.test.spec.js

```diff
@@ -1,41 +1,51 @@
 //https://127.0.0.1:3000/search?f=doctype%3ABOOK
 
 describe('frontsite literature details', () => {
-  const goToSearchDocument = () => {
-    // Books + available for loan
+  Cypress.Cookies.defaults({
+    preserve: 'csrftoken',
```

Comment:
This is to keep the CSRF token after visiting the login page once in order to avoid visiting it each time; am I right?

Reply:
Yes, it's because cypress cleans all the cookies before each test, and the login command has to use the csrf cookie in the request headers.